simple-chat-back/package.json

```diff
@@ -8,13 +8,15 @@
         "dev": "nodemon app.js",
         "audit": "audit-ci --high",
         "test": "jest",
-        "coverage": "npm test -- --coverage --watchAll=false"
+        "coverage": "npm test -- --coverage --watchAll=false",
+        "inspect": "jsinspect -t 20 ./src || true"
```

Comment:
I've read the documentation and the `|| true` makes the core to never fail the build, please remove this from this file and the package.json from the frontend as we want to fail the build if the inspection fails.

Reply:
Ready